<launch>
	<arg name="obstacle_config_file"
		 default="$(find obstacle_estimator)/config/obstacle.yaml" />

  	<node 	pkg="obstacle_estimator"
    		type="obstacle_filter_node"
    		name="obstacle_filter_node"
    		respawn="false"
    		launch-prefix=""
    		required="true">
		<rosparam file="$(arg obstacle_config_file)" command="load" />
  	</node>

</launch>
